### ISSUE TITLE
Website refresh & domain

### ISSUE THREAD

**Issue by chipsenkbeil** [vimwiki/vimwiki#1076]
Currently, vimwiki has a fairly barebones website in the form of vimwiki.github.io. Has there been interest in a more formal domain name? The wiki tld exists, although https://vim.wiki is taken. We could get something like https://vimwiki.wiki as an option.

I'm happy to pay for it if there's a good way for the ownership to be shared with others so it isn't just locked to me.

### Refresh on website

Additionally, would there be interest in a refresh of style/content, maybe something similar to https://orgmode.org/? I take a lot of inspiration to how other similar offerings compare. I've got a few separate extensions in the works to support vimwiki:

- [cobalt.rs](https://github.com/cobalt-org/cobalt.rs/issues/817) (static site generator) through a direct contribution to extend input language to support vimwiki
-  [mdbook](https://github.com/rust-lang/mdBook/issues/1183#issuecomment-753493074) (documentation site generator) through a new preprocessor to go from vimwiki -> markdown 
- ~~[Zola](https://github.com/getzola/zola/issues/1276) (static site generator) doesn't seem interested in a contribution at the moment~~

Of course, we could stick with generating the HTML directly from vimwiki (the plugin) as well.

**Comment by tinmarino** [comment]
I agree for the style. A Custom css and sooe class in the vimwiki seems nice.
For the domain name, I prefer to cencentrate on the content first: vimwiki.github.io is not ready to be on a permanent paid domain (i.e. currently overkill).
A Css and wiki2html collection can be nice.

**Comment by heitorPB** [comment]
I think a refresh on the website should come before a new domain.

Besides a new visual, what should be in the website? The things that came to my mind are:
- what is vimwiki
- main features
- how to install (in vim and neovim, using package managers, and with vim-packages)
- how to use the vimwiki syntax
- how to use the markdown syntax

And some questions:
1. should the `:help vimwiki` content be also in the website?
1. should merge [vimwiki's wiki](https://vimwiki.github.io/vimwikiwiki/)? Or keep it separated? Or move the website there?

**Comment by tinmarino** [comment]
I agree with @heitorPB that the content of the documentation site is more important that the style or the url.
I do use this page wiki or the vim doc and there is a site that is to be maintained at https://vimwiki.github.io/.
In my opinion it is more than enough considering no one is maintaining it.
